weave

```diff
@@ -1393,51 +1425,17 @@ launch_router() {
         --http-addr $HTTP_ADDR \
         --status-addr $STATUS_ADDR \
         --resolv-conf "/var/run/weave/etc/$RESOLV_CONF_BASE" \
-        "$@")
+        $PROXY_ARGS)
     wait_for_status $CONTAINER_NAME http_call $HTTP_ADDR
+    # Wait for Weave Proxy:
+    [ -n "$NO_PROXY" ] || wait_for_status $CONTAINER_NAME http_call_unix $CONTAINER_NAME status.sock
```

Comment:
Done in `fa9b1a5c4b61 TO SQUASH: start proxy before router, remove SignalHandlerLoop on proxy, do not sync on proxy status.`